src/platform/test/functional/page_objects/share_page.ts

```diff
@@ -42,7 +50,7 @@ export class SharePageObject extends FtrService {
   }
 
   async clickShareTopNavButton() {
-    return this.testSubjects.click('shareTopNavButton');
+    await this.testSubjects.click('shareTopNavButton');
```

Comment:
```suggestion
    return this.testSubjects.click('shareTopNavButton');
```

It was a leftover after some other changes which I reverted. Thanks!